src/project/project.controller.ts

```diff
@@ -47,11 +49,30 @@ export class ProjectController {
     ) {}
 
     @Get('/shared')
+    @ApiOperation({ summary: 'Throws an HTTP 418 error as a joke' })
```

Comment:
Saquemos que es joda por las dudas (?
```suggestion
    @ApiOperation({ summary: 'Throws an HTTP 418 error' })
```